peerdid/storage.py

```diff
@@ -0,0 +1,25 @@
+import os
+from abc import ABC, abstractmethod
+
+
+class Storage(ABC):
+    @abstractmethod
+    def save(self):
+        pass
+
+    @abstractmethod
+    def load(self):
+        pass
+
+
+class FileStorage(Storage):
+    def __init__(self, peer_did_storage_folder=os.path.expanduser('~/.peerdids'),
```

Comment:
It would be better to not call a method when default value is calculated, and rather have something like
```
def __init__(self, peer_did_storage_folder=None, ....):
    self._peer_did_storage_folder = peer_did_storage_folder or os.path.expanduser('~/.peerdids')
```